platform/elrond/model.go

```diff
@@ -47,6 +59,14 @@ func (tx *Transaction) TxStatus() blockatlas.Status {
 	}
 }
 
+func (tx *Transaction) Fee() blockatlas.Amount {
```

Comment:
Let's not compute this here, in blockatlas. At least in proxy (if not in node).

Reply:
done.